### DISCUSSION TITLE
FAQ - PostgresML

### DISCUSSION THREAD

**Discussion by giscus** [postgresml/postgresml#519] [Announcements]
# FAQ - PostgresML

Train and deploy models to make online predictions using only SQL, with an open source Postgres extension.

https://postgresml.org/about/faq/

**Comment by dusanmarjanovic** [comment]
"In addition, PostgresML automatically applies best practices for data cleaning like imputing missing values by default and normalizing data to prevent common problems in production. " 
I'm trying to use a dataset with some missing values, so how can this be achieved? thanks

**Reply by montanalow** [reply]
I've added some links to those docs in #521. You can read more details about imputing missing values here: 

https://postgresml.org/user_guides/training/preprocessing/
